src/components/tilt3d/Tilt3dDirective.js

```diff
@@ -0,0 +1,39 @@
+goog.provide('ga_tilt3d_directive');
+
+(function() {
+
+  var module = angular.module('ga_tilt3d_directive', []);
+
+  module.directive('gaTilt3d', function(gaBrowserSniffer) {
+    return {
+      restrict: 'A',
+      templateUrl: 'components/tilt3d/partials/tilt3d.html',
+      link: function(scope, element, attrs) {
+
+        scope.supported = gaBrowserSniffer.webgl;
+
+        // FIXME: remove this
+        // true is the selected background layer is not 3d compatible.
+        scope.disabled = false;
+        scope.$on('gaBgChange', function(evt, value) {
+          scope.disabled = value.id == 'ch.swisstopo.swissimage';
```

Comment:
`disable3d` property is only available in the background selector directive, not here